.github/ci-scripts/bash/functions-for-build

```diff
@@ -565,6 +575,64 @@ setup_qa_env() {
   deactivate
 }
 
+setup_packer_env() {
+  sudo python3 -m pip uninstall -y packer_framework
+  python3 -m pip install --user virtualenv --trusted-host ${NEXUS_SERVER_ENDPOINT}
+  python3 -m venv pkenv
+  source pkenv/bin/activate
+  python3 -m pip install wheel --trusted-host ${NEXUS_SERVER_ENDPOINT}
```

Comment:
Do we need `wheel`. It should already be present

Reply:
it also get's installed in setup_qa function https://github.com/TessellDevelopment/convoy-ci/pull/362/files#diff-b19e1fa8e74fc16a4d83587186447ccc27042ada73eca4a7bc45c6dfc5456193R588

Reply:
OK, did you try removing it and does it fail?

Reply:
I don't think wheel is present https://github.com/TessellDevelopment/ci-test-repo-b/actions/runs/11101547011/job/30839523591#step:7:425

It isn't failing as such (the commands are getting installed) but it's showing error when installing https://github.com/TessellDevelopment/ci-test-repo-b/actions/runs/11101452008/job/30839219605#step:7:598